### ISSUE TITLE
Missing required field indicator text

### ISSUE THREAD

**Issue by erangatl** [wso2/product-apim#11134]
### Description:

All the fields are required
![image](https://user-images.githubusercontent.com/20179540/116363975-55b14480-a821-11eb-94b6-b6f3b49ffcae.png)

### Environment details (with versions):
- OS:
- Client:
- Env (Docker/K8s):

---
